lib/index.ts

```diff
@@ -23,15 +23,9 @@ export const useStoryblokBridge = <
   cb: (newStory: ISbStoryData<T>) => void,
   options: StoryblokBridgeConfigV2 = {}
 ) => {
-  if (typeof window === "undefined") {
-    return;
-  }
-
-  if (typeof window.storyblokRegisterEvent === "undefined") {
-    console.error(
-      "Storyblok Bridge is disabled. Please enable it to use it. Read https://github.com/storyblok/storyblok-js"
-    );
-
+  const isServer = typeof window === "undefined";
+  const isBridgeLoaded = typeof window.storyblokRegisterEvent !== "undefined";
+  if (isServer || (!isServer && !isBridgeLoaded)) {
```

Comment:
Maybe you could simplify this and just check for `isBridgeLoaded` because if it's server-side the bridge won't be loaded, so that case is implicit